core/util.ts

```diff
@@ -80,3 +80,45 @@ export function isUntitledDocument(filename: string): boolean {
   // In vscode, tsserver may crash because a temporary document is not saved
   return /^untitled:/.test(filename);
 }
+
+/**
+ * find module which has no extension name
+ * We hope it can find the module correctly in Deno's way
+ * eg. `import { foo } from "./bar"` should be `import { foo } from "./bar.ts"`
+ * @param filepath
+ * @param moduleName
+ */
+export function findNonExtensionModule(
+  filepath: string,
+  moduleName: string
+): string {
+  function resolveModule(modulePath: string): string | void {
+    if (pathExistsSync(path.resolve(path.dirname(filepath), modulePath))) {
+      return modulePath;
+    }
+
+    return;
+  }
+
+  const denoSupportedExtensions = [
+    ".ts",
+    ".tsx",
+    ".d.ts",
+    ".js",
+    ".jsx",
+    ".mjs",
+    ".wasm",
```

Comment:
Deno does not actually support importing `.wasm` right now. Can you remove this?